assets/src/edit-story/components/library/panes/text/textSets/getTextSets.js

```diff
@@ -0,0 +1,76 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * Internal dependencies
+ */
+import { migrate } from '../../../../../migration/migrate';
+
+async function loadTextSet(name) {
+  const data = await import(`./raw/${name}.json`);
+  const migrated = migrate(data, data.version);
+
+  const textSets = migrated.pages.reduce((sets, page) => {
+    let minX = Infinity;
+    let maxX = 0;
+
+    let minY = Infinity;
+    let maxY = 0;
+
+    const textElements = page.elements.reduce((acc, element) => {
+      if (!element.isBackground) {
```

Comment:
Maybe use this as a condition for `page.elements.filter()`?

Reply:
I did a little reworking and changed it to a filter, I think it reads a little easier now.